prime-router/docs/how-to-setup-an-apple-silicon-mac.md

```diff
@@ -0,0 +1,91 @@
+# Setup Apple Silicon Macs for Development
+
+## Problems
+
+At the time of writing this note, Microsoft's Azure Function docker container is not compatible with Apple Silicon processors. 
+Microsoft has not announced plans to fix this problem. 
+
+The current `./devenv-infrastructure.sh` runs the Frontend build in a container. 
+This container doesn't run because of the use of ARM64. 
+
+## Workarounds
+
+Fortunately, most of our tooling works on Apple Silicon, and it is possible to run ReportStream without a Docker container.  
+
+### Step 1 - Follow the getting started instructions
+
+Follow most of the [Getting Started](getting_started.md) instructions, including:
+
+1. Installing the recommend tools 
+2. Run `./cleanslate.sh`
+
+The step that runs the `./devenv-infrastructure.sh` will not work. 
+
+### Step 2 - Run dependencies
+
+ReportStream depends on set of containers to be up before running. Run these containers now using `docker-compose`.
+
+```bash
+docker-compose up sftp redox azurite vault web_receiver --detach
+```
+
```

Comment:
I needed to update some environment settings before this. Pointing at localhost:10001 for azurite for example.

```
AzureWebJobsStorage="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1;QueueEndpoint=http://localhost:10001/devstoreaccount1;"
PartnerStorage="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1;QueueEndpoint=http://localhost:10001/devstoreaccount1;"
POSTGRES_USER=prime
POSTGRES_PASSWORD=changeIT!
POSTGRES_URL=jdbc:postgresql://localhost:5432/prime_data_hub
PRIME_ENVIRONMENT=local
OKTA_baseUrl=hhs-prime.okta.com
OKTA_clientId=0oa6fm8j4G1xfrthd4h6
OKTA_redirect=http://localhost:7071/api/download
VAULT_API_ADDR=http://localhost:8200
```